### ISSUE TITLE
Letting pyCaret work on its own

### ISSUE THREAD

**Issue by webzest** [pycaret/pycaret#652]
Hi,

Thank you for your amazing work and effort.  I wanted to enhance my prediction a bit, after using several approaches (stacking, ensemble, GridSearchCV, and HyperparameterTuning) to try to improve my results.  I got to a point where my tuning efforts were not changing my prediction accuracy.  so, I decided to give pyCaret a chance.  I ran my clean data through it using all 21 models to see what happens.  It took 3 days to finish (WOW!!).  the first time I tried, I had a power outage on Day 2, so I had to start over  :(.  At any rate, it finally finished and I examined the results.

This is what I found:
1. Compared the 21 Models:
```

                        Model	MAE	MSE	RMSE	R2	RMSLE	MAPE	TT (Sec)
0	CatBoost Regressor	0.0831	0.0164	0.1258	0.8946	0.0097	0.007	11.52
1	Huber Regressor	0.0859	0.0185	0.1312	0.8806	0.0101	0.0072	0.6678
2	Light Gradient Boosting Machine	0.0915	0.0187	0.1349	0.8788	0.0104	0.0076	0.5643
3	Gradient Boosting Regressor	0.0927	0.0193	0.1367	0.8751	0.0105	0.0078	0.9747
4	Bayesian Ridge	0.0902	0.0201	0.1376	0.8688	0.0106	0.0075	0.1977
5	Ridge Regression	0.0941	0.0218	0.1433	0.8566	0.011	0.0079	0.0201
6	Passive Aggressive Regressor	0.0996	0.022	0.1454	0.8561	0.0112	0.0083	0.065
7	Extra Trees Regressor	0.1009	0.0224	0.148	0.8538	0.0115	0.0084	0.6785
8	Random Forest	0.101	0.0227	0.1487	0.8536	0.0115	0.0084	0.6468
9	Extreme Gradient Boosting	0.1028	0.0238	0.1523	0.8452	0.0118	0.0086	0.5894
10	Support Vector Machine	0.1065	0.0251	0.1567	0.8391	0.0121	0.0089	0.4006
11	TheilSen Regressor	0.0941	0.0258	0.1521	0.83	0.0116	0.0079	501.1
12	Orthogonal Matching Pursuit	0.0934	0.0283	0.1596	0.8079	0.0123	0.0078	0.0347
13	AdaBoost Regressor	0.1371	0.0336	0.1817	0.7829	0.014	0.0114	0.7373
14	Decision Tree	0.1471	0.044	0.2072	0.7129	0.016	0.0123	0.0814
15	K Neighbors Regressor	0.1708	0.0551	0.2335	0.6425	0.0179	0.0142	0.0485
16	Elastic Net	0.2568	0.1137	0.3355	0.2674	0.0257	0.0213	0.0182
17	Lasso Regression	0.3057	0.1546	0.3918	-0.0013	0.0301	0.0254	0.0152
18	Lasso Least Angle Regression	0.3066	0.1554	0.3929	-0.0069	0.0301	0.0255	0.0173
19	Linear Regression	3.085e+59	6.464e+121	3.097e+60	-4.04e+122	13.96	2.639e+58	0.1285
20	Random Sample Consensus	7.642e+60	5.669e+124	7.716e+61	-3.774e+125	18.52	6.295e+59	9.315
21	Least Angle Regression	8.512e+104	7.39e+212	8.597e+105	-4.616e+213	37.87	7.253e+103	0.2597


```
Then, I tuned , Stacked, and Finalized Them them:

Finally, I predicted on my unseen data...

My results were unuseable....  They were all off and I measured the results on kaggle, which generated a score of 0.43122.  That is at the bottom of the pile...  

I love your concept and think it is a GREAT approach to running ML.  How would you recommend I approach pyCaret with my project and goals of getting the most accurate prediction possible?

By the way, I am using the Kaggle Housing Price (advance) Data Set (Train and Test)

**Comment by webzest** [comment]
@Yard1 : Do you mean that I need to create just one model, catboost, untuned as a prediction starting point?  
Then, what is next?

**Comment by Yard1** [comment]
Then see what results it gives on kaggle :P

**Comment by E-B-Manohar** [comment]
Any Luck @webzest ?

**Comment by webzest** [comment]
I was able to clean the data, as you indicated, and fed it to pyCaret.  Unfortunately, the best I can do is a 0.12x, wich is still too high, but it is better than a 0.14.  Is there a way to use pyCaret to fine tune the estimators parameters to enhance the model training?

**Comment by Yard1** [comment]
`tune_model` function. You can also pass your own tune grids. In the currently released version (2.1), only random search is supported.

**Comment by pycaret** [comment]
I am closing the issue as it's not related PyCaret specifically.
